go.mod

```diff
@@ -22,6 +22,8 @@ require (
 	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/juju/fslock v0.0.0-20160525022230-4d5c94c67b4b
 	github.com/klauspost/compress v1.13.1 // indirect
+	github.com/lib/pq v1.10.2
```

Comment:
License?

Reply:
removed postgresql support for now, figure it out later.